### ISSUE TITLE
Creating issues to update alt texts on Events page

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#3132]
### Dependency
- [x] #3255
- [x] #3281
- [x] #3279 
- [x] #3280 
- [x] #3282 
- [x] #3286
- [x] #3300 
- [x] #3308 
- [x] #3316 
- [x] #3307 
- [x] #3310
- [x] #3314
- [x] #3309 
- [x] #3312 
- [x] #3315
- [x] #3313 
- [x] #3311
- [x] #3284
- [x] #3278 
- [x] #3306 

### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this epic, we will write issues to change the alt text for the images in the [Events page](https://www.hackforla.org/events/) which will then be added to dependency items above.

### Details
Designers and PMs identified that some alt texts in the [Events page](https://www.hackforla.org/events/) should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] This issue should only be assigned to a lead.
- [x] Lead would guide other developers on creating issues using the following steps (These steps would be recurring for each issue that would be created and this action item should stay unchecked until all the issues have been created):
    - Check the following [Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=253767243) to track the issue creation progress (there are 19 issues in total that need to be made)
    - Choose a row for which issue needs to be created and add your name in the Issue writer column. (If multiple team members creating issues simultaneously, let other team members know which row you would be working on, so there are no duplicate issues).  See column H to know if an issue has been made already
    - Create a new issue with steps to change the alt text for an image following the sample issue below as well as using [How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki.
    - Update the issue link in the [Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=253767243)
- [x] Add the issue numbers of all the issues that have been created to the dependency section above.
- [x] If all the issues for the spreadsheet tab have been created, add a dependency label and move this issue to ice box column.
- [x] If all the dependency issues are closed, remove the dependency label, move this issue from ice box to In progress column and close this issue.

### Resources/Instructions
[Events page](https://www.hackforla.org/events/)
[Google spreadsheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=253767243) to track issue creation
[How to create issues](https://github.com/hackforla/website/wiki/How-to-create-issues) wiki

### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will change the alt text for the westside location on [Events page](https://www.hackforla.org/events/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [ ] For the file `_includes/events-page/our-locations-content.html`, change [line 5](https://github.com/hackforla/website/blob/c88d715eeaca83a1665cfce48be1bead3f203fde/_includes/events-page/our-locations-content.html#L5) for the alt text:

from
```
<img class="location-image" src="../assets/images/hack-nights/westside1.png" alt="westside" />
```
to
```
<img class="location-image" src="../assets/images/hack-nights/westside1.png" alt="Westside" />
```
- [ ] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3132 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3132 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3132 
- [Events page](https://www.hackforla.org/events/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/events-page/our-locations-content.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/c88d715eeaca83a1665cfce48be1bead3f203fde/_includes/events-page/our-locations-content.html#L5

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by JessicaLucindaCheng** [comment]
- I pulled several issues from the Dependency section and closed them as unplanned because changes in #3027 made those issues' alt text no longer valid since their associated images no longer exist on the page.

**Comment by JessicaLucindaCheng** [comment]
Sorry, I accidentally closed it when I was trying to just leave a comment.

**Comment by jdingeman** [comment]
All dependencies have been checked off. Moved to New Issue Approval and removed Dependency label per 
- #3255

**Comment by JessicaLucindaCheng** [comment]
Closed this issue per the last action item in this issue.
